src/src.md

```diff
@@ -0,0 +1,3 @@
+### SRC
```

Comment:
This file is completely useless, you use python documentstion to explain code structure, not md files